File: sort_associative_array_by_key_descend/sort_associative_array_by_key_descend.py
example_array = {1: "January", 12:"December", 3:"March" }
tuples_sorted_by_key = sorted(example_array.items(), key=lambda x:x[0])[::-1]
for key, value in tuples_sorted_by_key:
  print(str(key) + " => " + value)

File: examples/sort_associative_array_by_value_descend/sort_associative_array_by_key_descend.js
const example_array = {"January":1, "December":12, "March":3};
var keys=[];
for(var key in example_array) {
  keys.push(key);
}
keys.sort((a,b)=>{return example_array[b]-example_array[a];});
for(var key of keys) {
  console.log(key + " => " + example_array[key] );
}